src/parsers/pm2ps_parser.jl

```diff
@@ -6,7 +6,7 @@ Takes a dictionary parsed by PowerModels and returns a PowerSystems
 dictionary.  Currently Supports MATPOWER and PSSE data files parsed by
 PowerModels
 """
-function pm2ps_dict(data::Dict{String,Any})
+function pm2ps_dict(data::Dict{String,Any}, genmap_file::Union{String,Nothing}=nothing)
```

Comment:
I would like to discuss the intended usage of genmap_file. It feels a bit awkward to pass this filename throughout the call stack?

Reply:
It seems like the user should be able to pass this config filename to the PowerSystem constructor and the location can be stored in memory.  Other places in the code that need it can look it up.  Then you wouldn't need to pass it into any functions.

Reply:
Now can be passed as a keyword argument, per our discussion.